README.md

```diff
@@ -19,7 +19,7 @@ defmodule Posts do
   use Ecto.Filters, key: :search
   alias MyProject.{Post, Repo}
 
-  add_filter(:comment_body, fn value, query -> 
+  filter(:comment_body, fn value, query ->
```

Comment:
Since the tests have been changed from `posts` and `comments` to `users`. It might make sense to have these examples reflect what is being done in there as well? This would probably serve as a better example since they can dig into the tests and see how it all works too.